### DISCUSSION TITLE
`body-till`

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1609] [Ideas & RFCs]
```
-- fastn.get.description:
$body-till$: EOB

-- import: fastn/processors as pr

-- string http-location:
$processor$: pr.test-data

-- ds.button: $http-location

EOB
```

Or even better:

```
-- fastn.get.description:

<EOB
-- import: fastn/processors as pr

-- string http-location:
$processor$: pr.test-data

-- ds.button: $http-location

EOB>
```

```
-- fastn.get.description:
code: <CODE

-- import: fastn/processors as pr

-- string http-location:
$processor$: pr.test-data

-- ds.button: $http-location

CODE>
```

How we parse content inside `<CODE` till `CODE>` will depend on type of `code` in `fastn.get.description`.

**Comment by siddhantk232** [comment]
What is the context here? We're considering `<CODE` till `CODE>` syntax as an alternative to the `-- end: code` syntax?

**Reply by amitu** [reply]
This is trying to solve the escpaing problem in code blocks. 

I am not too fond of this tho. We can do it, it does not conflict with other discussions, but it may be too much.
